cmd/fetchbranches/main.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+func main() {
+	ci := NewCI()
+
+	if ci == nil {
+		fmt.Fprintf(os.Stderr, "CI not found\n")
+		os.Exit(1)
+	}
+
+	// Shows PR number
+	pr, err := ci.GetPR()
+
+	if err != nil {
+		fmt.Printf("%s\n", err)
```

Comment:
Just a general point. It's often useful to use a set of standard prefixes for all logging messages. For example, for all the `Printf()` calls in this function, I'd be tempted to do something like:

```
fmt.Printf("ERROR: %v\n", err)
```

Alternatively, you could use a logging library like https://godoc.org/github.com/sirupsen/logrus which will add in the appropriate tags based on the log level. These prefixes make the output much more consistent and  "consumable".